### DISCUSSION TITLE
Swedish translation of Sparkle

### DISCUSSION THREAD

**Discussion by be-born** [sparkle-project/Sparkle#2592] [Q&A]
Hi all,
i'm the Swedish translator/localizer of GraphicConverter, which is one of the apps that uses Sparkle for updates. I wonder how I shall proceed if I want to make a swedish localization of Sparkle?

It seems to be needed to fully localize GraphicConverter.

Best, 

/Bertil

**Comment by zorgiepoo** [comment]
Sparkle already has Swedish translations. If you'd like to make updates to them (e.g. maybe some strings are missing or some text needs to be revised) feel free to create a pull request. See https://github.com/sparkle-project/Sparkle/pull/2573/ or [any other recent localization updates](https://github.com/sparkle-project/Sparkle/pulls?q=is%3Apr+is%3Aclosed+localization) people have contributed as examples.
